### PR TITLE
Improving Region Availability

### PR DESCRIPTION
**Pull request by marvinbuss**
**This PR fixes**
Improving Region Availability for DMA by including the list of regions where ZRS is available. Also removed unnecessary dependency based on linting rules. This will fix: https://github.com/Azure/data-management-zone/issues/256

### CODE REVIEW

**Review by marvinbuss — COMMENTED** [review]
LGTM!

**Review by marvinbuss — COMMENTED** [review]
LGTM!

**Review by marvinbuss — COMMENTED** [review]
LGTM!

**Comment by marvinbuss** [comment]
[Preview Link](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fdata-landing-zone%2Fmarvinbuss%2Flocation_improvements%2Finfra%2Fmain.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fdata-landing-zone%2Fmarvinbuss%2Flocation_improvements%2Fdocs%2Freference%2Fportal.dataLandingZone.json)

File: docs/DataManagementAnalytics-Prerequisites.md [hunk 1]
```diff
@@ -48,22 +48,27 @@ To help you more quickly understand the structure of the repository, here is an
 For now, we are recommending to select one of the regions mentioned below. The list of regions is limited for now due to the fact that not all services and features are available in all regions. This is mostly related to the fact that we are recommending to leverage at least the zone-redundant storage replication option for all your central Data Lakes in the Data Landing Zones. Since zone-redundant storage is not available in all regions, we are limiting the regions in the Deploy to Azure experience. If you are planning to deploy the Data Management Zone and Data Landing Zone to a region that is not listed below, then please change the setting in the corresponding bicep files in this repository. Officially supported regions are:
```
Comment (amanjeetsingh) [inline docs/DataManagementAnalytics-Prerequisites.md:48]: Should we change "officially supported" ? I mean we do not want to restrict adoption of DMA to regions where let's say AZs are available. As a parttern, DMA is composed of services which are available out of most regions. Using word such as "Support" could be misinterpreted as MSFT's support for various services. 

Perhaps, "DMA deployment has been tested in the following regions:"

**Review by amanjeetsingh — COMMENTED** [review]
Proposed minor changes to verbiage around support and coverage. Thanks.

File: docs/DataManagementAnalytics-Prerequisites.md (hunk 1 above)
Comment (marvinbuss) [reply docs/DataManagementAnalytics-Prerequisites.md]: Thanks for the suggestion. Updated the wording.

**Review by marvinbuss — COMMENTED** [review]
LGTM!

**Review by esbran — DISMISSED** [review]
Looks good

**Review by esbran — APPROVED** [review]
Merry Christmas

**Pull request merged**